### DISCUSSION TITLE
Github Student for Concordia University, Montreal

### DISCUSSION THREAD

**Discussion by zackwalton** [community/community#50871] [GitHub Education] [answered]
I am a CS student at Concordia University, Montreal with a school email in the form of *********@live.concordia.ca. It says the following when reverifying my information (because I recently got I email I need to reverify, I assume it is this change that is making me do so).

Here is the error message:
![image](https://user-images.githubusercontent.com/24999242/227389016-0a4a6243-5c5f-4aa5-8090-ad5c67f9996d.png)

The issue is I don't think every student has either of the following:
1. a **csu.qc.ca** email (Concordia Student Union)
2. a **econcordia.com** email (asynchronous class website like google classroom or Moodle)

The live.concordia.ca is the Microsoft-controlled account we all get, is it possible to add it to the list of emails for Concordia University, Montreal?

**Comment by samus-aran** [comment, marked as answer]
Hi @zackwalton 👋 

Thank you for bringing this to our attention! We have updated this school's details and ask you to please re-apply. Let us know how you get on 👍
